src/App.js

```diff
@@ -180,6 +180,20 @@ export async function decodeFSH(encodedFSH) {
   }
 }
 
+async function getGistContent(gistId) {
+  const gistContent = await fetch(`https://api.github.com/gists/${gistId}`).then((res) => res.json());
+  let fshContent = '';
+  for (let f of Object.values(gistContent?.files || {})) {
+    if (f.truncated) {
+      fshContent += await fetch(f.raw_url).then((res) => res.text());
+    } else {
+      fshContent += f.content;
+    }
+    fshContent += '\n';
```

Comment:
Should there be two new lines between separate files? If I have a FSH entity in each file, I think we want them to be separated by an extra new line, right?

Reply:
Forgot to do this in https://github.com/FSHSchool/FSHOnline/commit/0780e82215ff0c75c62077db37ffa38348f66519.

Reply:
But remembered to do it in https://github.com/FSHSchool/FSHOnline/pull/70/commits/f148a0fd66f27a5025d7a183519a2ed6c6c02fa9.